dbt/ui/printer.py

```diff
@@ -250,16 +244,20 @@ def print_run_result_error(result):
             sql_text = f.read()
             f.close()
 
-        rollbar_sql = ("Compiled SQL: \n\n{}\n".format(sql_text))
-        logger_sql = ("Compiled SQL: " + cyan("\n\n{}\n".format(sql_text)))
-
-        error_message = error_path + "\n" + error_result + "\n\n" + rollbar_sql
-        rollbar.report_message(message=error_message)
-
-        logger.info(yellow(error_path))
-        logger.info(error_result)
-        logger.info("")
-        logger.info(logger_sql)
+        rollbar_token = os.environ.get('ROLLBAR_ACCESS_TOKEN')
+        rollbar_env = os.environ.get('ENV_NAME')
+
+        if rollbar_token and rollbar_env:
+            rollbar.init(rollbar_token, rollbar_env)
+            rollbar_sql = ("Compiled SQL: \n\n{}\n".format(sql_text))
+            error_msg = error_path + "\n" + error_result + "\n\n" + rollbar_sql
+            rollbar.report_message(message=error_msg)
+        else:
```

Comment:
I would say we still want to do everything below here whether or not rollbar is init'd